allennlp/modules/seq2seq_encoders/stacked_self_attention.py

```diff
@@ -129,13 +129,11 @@ def forward(self, inputs: torch.Tensor, mask: torch.Tensor): # pylint: disable=a
             output = add_positional_features(inputs)
         else:
             output = inputs
-        for (attention,
-             feedforward,
-             feedforward_layer_norm,
-             layer_norm) in zip(self._attention_layers,
-                                self._feedfoward_layers,
-                                self._feed_forward_layer_norm_layers,
-                                self._layer_norm_layers):
+        for i in range(len(self._attention_layers)):
```

Comment:
Can you add a comment here that it's preferable to just use `ModuleList`, but since the original code didn't do that, we use this `getattr` hack to make the code compatible with multiple GPUs without breaking existing models?  We don't want someone to copy this, because `ModuleList` would have been better.

Reply:
Excellent point. I added a comment here and near the `add_module` calls. Thanks for the review!